.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
```

Comment:
- In the `Create release notes` job, consider providing more context in the comments to explain the purpose and flow of the code.